### PR TITLE
Add favicon based on d-marque asset

### PR DESCRIPTION
**Pull request by snim2**

### CODE REVIEW

File: src/_layouts/default.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-
+    <link rel="shortcut icon" type="image/x-icon" href="/assets/favicon.ico?">
```
Comment (yndajas) [inline src/_layouts/default.html:9]: Is the `?` in the `href` meant to be there?

**Review by yndajas — APPROVED** [review]
At last! 🎉

I wonder if it's worth adding a full favicon/PWA icon suite, e.g. the ones that are generated by [realfavicongenerator.net](https://realfavicongenerator.net)? That gives better support for different platforms and non-traditional browser usage, though that's likely an edge case for this repo

File: src/_layouts/default.html (hunk 1 above)
Comment (snim2) [reply src/_layouts/default.html]: No, good catch!

File: src/assets/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            "type": "image/png"
+        },
+        {
+            "src": "/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
```
Comment (yndajas) [inline src/assets/site.webmanifest:17]: Likewise here

**Pull request merged**